tests/zfs-tests/tests/functional/fault/fault.cfg

```diff
@@ -30,13 +30,19 @@ verify_runnable "global"
 export DISK_ARRAY_NUM=$($ECHO ${DISKS} | $NAWK '{print NF}')
 export DISKSARRAY=$DISKS
 export FSIZE=10M
-export MAXTIMEOUT=20
+export MAXTIMEOUT=600
```

Comment:
OK, that's clearly not the issue.  Let's go back to 20s which does work with the existing version of the test case.